docs/Governance.md

```diff
@@ -119,30 +121,46 @@ Responsibilities:
 ## Project Milestones
 
 eBPF for Windows uses monthly milestones in the format `YYMM`. So, e.g., the milestone for June, 2024 will be called 2406.
-At any time, there are four active milestones in the project. There will be a milestone for the current calendar month (`N`),
-and two others for the next two months (`N+1`), (`N+2`). And there is always a `Backlog` milestone for backlog issues to be
-addressed in the future. In the first triage meeting of a month, the previous milestone must be closed and a new `N+2` milestone must be created.
-
-During the weekly triage meetings, whenever any issue is marked with the "triaged" label, triage drivers MUST set an issue to one of the four milestones
-according to the priority of the issues.
+At any time, there will be one current milestone (for the current month) and one or more future milestones, in the project.
+A `Backlog` milestone exists for backlog issues.
+In the weekly triage meetings, new issues should be added to a milestone following guidelines in the
+[Prirotizing Issues](Governance.md#prioritizing-issues) section.
 
 ## Prioritizing Issues
 
 To help prioritize issues, the triage drivers and maintainers MUST use *special labels* to prioritize issues as follows:
 ### P1:
 This label represents highest priority. This label should be used for bugs impacting users, new features, or enhancements that are urgently
-requested by users. `P1` issues MUST be set to the *current milestone (`N`)*. If there is not enough time left in the current milestone to address
-the `P1` issue, it must be set to the *next milestone (`N+1`)*.
+requested by users. `P1` issues MUST be set to the *current milestone*.
 ### P2:
-This label is used for the next level of priority. Issues marked as `P2` can be set to milestones `N`, `N+1` (preferred) or `N+2`.
+This label is used for the next level of priority. Issues marked as `P2` can be set to the current milestone or to a future milestones (preferred).
 ### P3:
-This label is used for low priority issues. Issues marked with `P3` label can be set to any milestone including `Backlog`.
+This label is used for low priority issues. Issues marked with `P3` label can be set to any future milestone or the `Backlog` milestone.
+
+## Milestone Issues Limit
+At most 20 issues can be added to a project milestone. This limit makes it manageable for contributors to address all issues
```

Comment:
I disagree with a fixed constant hard-coded here, as what is manageable for contributors varies by the number of active contributors, which varies over time.  E.g., during a holiday period it will be fewer and during say summer internships or a hackathon (if one is planned) it may be more.  It also assumes that all issues are equally difficult, which isn't true.

Also if some tool automatically files issues, say one per instance of some code issue that occurs 20 times, this would cause you to either reject other P1 issues, or go through all the work of combining the issues just to work around this limit.

Reply:
I have removed the fixed limit, and made it a soft limit based on capacity of contributors, complexity etc. But I have left a rule of thumb of not having more than (3 x contributors) issues per milestone.